.github/workflows/2-deploy-to-staging.yml

```diff
@@ -35,7 +35,7 @@ jobs:
           accountId: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
           projectName: interface-staging
           directory: build
-          githubToken: ${{ secrets.GITHUB_TOKEN }}
+          gitHubToken: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
revert?

Reply:
well this is an error i saw when i was looking at these files - see the PR description

Reply:
don't change these even if your IDE, as it's risky and it works as-is

Reply:
ok,, reverted these too!